docs/noencap-hybrid-modes.md

```diff
@@ -34,6 +34,13 @@ spec:
               value: "true"
 ```
 
+Note that changing the traffic mode in an existing cluster, where Antrea is
+currently installed or was previously installed, may require restarting existing
+workloads. In particular, the choice of traffic mode has an impact on the MTU
+value used for Pod network interfaces. When changing the traffic mode from
+`NoEncap` to `Encap`, existing workloads should be restarted, so that new
```

Comment:
Probably better to keep the words the same the options in antrea-agent.conf. 
```suggestion
`noEncap` to `encap`, existing workloads should be restarted, so that new
```

Reply:
The current version is consistent with the rest of the document. We can update the document as a whole in a separate PR (I don't want a large unrelated documentation change in this PR), but I am not sure it is necessary.

Reply:
Sure, no strong opinion for this.